src/Infrastructure/Persistence/Doctrine/Migrations/Version20240329102233.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240329102233 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('
```

Comment:
Petite suggestion d'indentation : 

```suggestion
        $this->addSql('
            INSERT INTO "user" (uuid, full_name, email, password) VALUES
                (\'13ad7a59-bd3c-4e1d-8f85-8b148ad00705\', \'Julien JACQUELINET\', \'julien.jacquelinet@beta.gouv.fr\', \'$2y$13$0vx4Anj.TVy8D0iGrER4zuKT//vYlnEx1bLpoUZnv2lNPmWNWXrQC\')
        ');
```